components/tikv_util/src/lru.rs

```diff
@@ -174,33 +178,82 @@ impl<K, V> SizePolicy<K, V> for CountTracker {
     }
 }
 
-pub struct LruCache<K, V, T = CountTracker>
+/// Some [`EvictPolicy`] may need to know what the entry bing popped out is to
```

Comment:
Better do desribe more details about the `Some` here so others could know where is it actually used by now.

Reply:
I'll add this description in the next PR.